### ISSUE TITLE
How to run the model without contrastive learning?

### ISSUE THREAD

**Issue by allanj** [zwx980624/mwp-cl#1]
as in the title.

**Comment by zwx980624** [comment]
try script like this
python3 run_ft.py \
    --output_dir output/mwp-ft-multilingual-en \
    --data_dir data \
    --train_file MathQA_mbert_token_train.json \
    --dev_file MathQA_mbert_token_val.json \
    --test_file MathQA_mbert_token_test.json \
    --n_val 1 --n_save_ckpt 10 --schedule linear --batch_size 16 --learning_rate 5e-5 --n_epochs 50 \
    --warmup_steps 4500 --hidden_size 768 --beam_size 3 --dropout 0.5 --seed 17
